APE24.rst

```diff
@@ -0,0 +1,497 @@
+------------------------------
+
+We do not think so! In the pre-APE 24 process, the reviewer was anonymous by default
+because we modeled the process after traditional astronomical journals.
+Even then, the reviewer had the option to reveal themselves if they wanted to.
+Given that the Astropy ecosystem is open-source anyway, it makes sense for us
+to move away from the journal-style tradition and embrace a more open process that
+pyOpenSci uses.
+
+Will we no longer need Editors?
+-------------------------------
+
+We still need them! However, instead of managing the whole process themselves,
+they will now be part of the `pyOpenSci Editorial Board`_ and perform their
+duties within the pyOpenSci process as laid out in this APE.
+
+Rejecting packages
+------------------
+
+A package may be "rejected" in different stages of the review, either by the
+pyOpenSci Editor-in-Chief or the editor, including but not limited to causes
+such as:
+
+* the package is not in technical or domain scope;
+* the package fails one of the pre-review checks done by the Editor-in-Chief;
+* the package maintainer stops responding to review comments; or
+* the package maintainer is unwilling to ensure the package can be used and maintained.
+
+While in practice, rejection is rare, a rejection does *not* have to come from
+the Astropy community directly and will be objective.
+
+A package that fails the Astropy-specific criteria may still be accepted into the
+pyOpenSci ecosystem (and also published by JOSS) if it is in scope;
+an example of this might be a well-documented and developed Python package that
+is useful to the scientific community that fails to relate well to the Astropy ecosystem.
+
+"Re-review" of packages
+-----------------------
+
+**Note: This does not exist yet and probably will not happen before
+this APE is accepted. As of July 2023, pyOpenSci had a somewhat
+manual process to check in on an accepted package.**
+
+One core goal of pyOpenSci is to support scientific Python packages that are
+maintained over time. Due to the lack of resources, it does not do a *full*
+re-review of packages (involving Editor-in-Chief, editor, and 2 reviewers).
+However, it has plans to set up automated checks that track the "health"
+and maintenance level of a package over time, with the goal of identifying
+packages that have become "orphans" (i.e, unmaintained). Once a package is
+identified as needing additional maintenance, it will be flagged.
+
+"Health" might include frequency of commits, releases, CI status, etc.
+Some, if not most, of this data will be collected using
+`devstats <https://github.com/scientific-python/devstats>`_ and
+`repo-review <https://github.com/scientific-python/repo-review>`_ from
+`Scientific Python`_. This information will be displayed on public dashboards
+hosted on the pyOpenSci website; we may also grab those same badges,
+where applicable, for a similar dashboard on Astropy website
+focused only in our Affiliated packages.
```

Comment:
@nabobalis , while this won't be immediately available, it is planned. Would such a plan make you less nervous about the current static badges going away?

Reply:
I have no personal issue with those going, I was just voicing the concerns from @Cadair. 

I think this plan is great and I support it 100%.